[CARBONDATA-3243] Updated DOC for No-Sort Compaction and a few Fixes (#3064)
apache/carbondata: docs/dml-of-carbondata.md

| [COMMENTCHAR](#commentchar)                             | Character used to comment the rows in the input csv file. Those rows will be skipped from processing |
| [HEADER](#header)                                       | Whether the input csv files have header row                  |
| [FILEHEADER](#fileheader)                               | If header is not present in the input csv, what is the column names to be used for data read from input csv |

> | [SORT_SCOPE](#sort_scope)                               | Sort Scope to be used for current load.                      |

Reviewer: ```suggestion
| [SORT_SCOPE](#sort_scope)                               | Sort Scope is used for current load.                      |
```

Author: "to be" is correct. No change.

    OPTIONS('FILEHEADER'='column1,column2') 
    ```

  - ##### SORT_SCOPE:

>   Sort Scope to be used for the current load. This overrides the Sort Scope of Table.

Reviewer: ```suggestion
  Sort Scope is used for the current load. This overrides the Sort Scope of Table.
```

Author: "to be" is correct. No change.

  ```
  CLEAN FILES FOR TABLE carbon_table
  ```

>   

Reviewer 2: revert this change

Author: Done.
